01week/rockPaperScissors.js

```diff
@@ -9,30 +9,39 @@ const rl = readline.createInterface({
 
 
 function rockPaperScissors(hand1, hand2) {
-  
+
   if (hand1 === "rock" && hand2 === "scissors") {
-   return 'hand1 wins'
+    return 'hand1 wins'
+  }
+
+  else if (hand1 === "paper" && hand2 === "rock") {
+    return 'hand1 wins'
   }
 
-  if (hand1 === "paper" && hand2 === "rock") {
+  else if (hand1 === 'scissors' && hand2 === 'paper') {
     return 'hand1 wins'
   }
 
-  if (hand1 === 'scissors' && hand2 === 'paper') {
-  return 'hand1 wins'
+  else if (hand1 === 'rock' && hand2 === 'paper') {
+    return 'hand2 wins'
+  }
+
+  else if (hand1 === 'paper' && hand2 === 'scissors') {
+    return 'hand2 wins'
+  }
+
+  else if (hand1 === 'scissors' && hand2 === 'rock') {
+    return 'hand2 wins'
   }
 
-  if (hand1 === 'paper' && hand2 === 'scissors') {
+  else if (hand1 === 'scissors' && hand2 === 'paper') {
     return 'hand2 wins'
```

Comment:
This statement is incorrect. if hand1 === 'scissors' && hand2 === 'paper' => hand1 wins!